### ISSUE TITLE
HttpException: , uri = https://xxxxxxxx/ URL 

### ISSUE THREAD

**Issue by fabio21** [flutterchina/dio#1192]
### New Issue Checklist

- [ ] I have searched for a similar issue in the [project](https://github.com/flutterchina/dio/issues) and found none

### Issue Info

| Info                           | Value                                                 |      |
| ------------------------------ | ----------------------------------------------------- | ---- |
| Platform Name                  | e.g. flutter / ios / android                          |      |
| Platform Version               | e.g.  1.5.0 / 12.0 / 9.0                              |      |
| Dio Version                    | e.g.  2.1.0 / 1.0.17                                  |      |
| Android Studio / Xcode Version | e.g. Android Studio 3.3.2 / Xcode 10.2.1              |      |
| Repro rate                     | e.g. all the time (100%) / sometimes x% / only once   |      |

used in another flutter project in 2019
and it worked, using the azure application service

But now only fear this error and not
constant
 
(HttpException: , uri = https://xxxxxxxx/ URL)

String _path = 'assets/cetificate.pfx';
Uint8List fileAas = (await rootBundle.load(_path)).buffer.asUint8List();
SecurityContext securityContext = SecurityContext();
securityContext..setTrustedCertificatesBytes(fileAas!, password: 'xxxx');
HttpClient httpClient = HttpClient(context: securityContext);

*** DioError ***:
I/flutter (19668): uri: https://xxxxxxxxxxxxxxxxxx/
I/flutter (19668): DioError [DioErrorType.other]: HttpException: , uri = https://xxxxxxxxxxxxxxxxxx/
I/flutter (19668): #0      DioMixin.assureDioError (package:dio/src/dio_mixin.dart:819:20)
I/flutter (19668): #1      DioMixin._dispatchRequest (package:dio/src/dio_mixin.dart:678:13)
I/flutter (19668): <asynchronous suspension>
I/flutter (19668): #2      DioMixin.fetch.<anonymous closure>.<anonymous closure> (package:dio/src/dio_mixin.dart)
I/flutter (19668): <asynchronous suspension>

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by damonZDM** [comment]
I also encountered the same problem frequently. I'm not sure whether it's the DIO problem or the server problem

**Comment by Lawati97** [comment]
@damonZDM Where you able to figure the problem?

**Comment by zengbo88** [comment]
same here
